### PR TITLE
docs(review): docs content review

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: tutorials/installation-jekyll/index.mdx [hunk 1]
```diff
@@ -9,15 +9,13 @@ tags: compute cms Jekyll Git Instance
 categories:
   - compute
 dates:
-  validation: 2023-05-24
+  validation: 2023-11-27
   posted: 2018-07-12
 ---
 
-Jekyll is a tool built on Ruby that transforms MarkDown files into static websites.
+Jekyll, a powerful tool built with Ruby, designed to convert MarkDown files into static websites. Unlike server-side languages such as PHP that generate a new page for each visitor, Jekyll takes a different approach. It transforms your MarkDown content into static HTML files, minimizing RAM and CPU usage. This method ensures that your web pages are pre-generated, providing a faster and resource-friendly solution.
```
Comment (jsudraud) [inline tutorials/installation-jekyll/index.mdx:16]:
```suggestion
Jekyll is a powerful tool built with Ruby, designed to convert MarkDown files into static websites. Unlike server-side languages such as PHP that generate a new page for each visitor, Jekyll takes a different approach. It transforms your MarkDown content into static HTML files, minimizing RAM and CPU usage. This method ensures that your web pages are pre-generated, providing a faster and resource-friendly solution. 
```

**Review by jsudraud — APPROVED** [review]

File: network/public-gateways/how-to/configure-a-public-gateway.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ This page shows you how to attach a [Public Gateway](/network/public-gateways/co
 
 3. Click the **Private Networks** tab. A list of Private Networks attached to the Public Gateway displays. If no Private Networks are attached, the list will be empty.
 
-4. Click <Icon name="plus" /> to attach a new Private Network to the Public Gateway.
+4. Click the "plus" icon (<Icon name="plus" />) to attach a new Private Network to the Public Gateway.
```
Comment (ldecarvalho-doc) [inline network/public-gateways/how-to/configure-a-public-gateway.mdx:33]: pas besoin d'ajouter ça je pense

File: tutorials/install-parse-server/index.mdx [hunk 3]
```diff
@@ -62,9 +62,14 @@ MongoDB is a document-oriented database that is free and open-source. It conside
 
 By default, Node.js is not available in the Ubuntu 20.04 default repository. Therefore Node.js needs to be added to the repository of your system. In addition, ensure `curl` is installed on your system.
 
-1. Add an external repository for the required version of NodeJS. Make sure to replace `lts` by the lastest node runtime.
+1. Add an external repository for the required version of NodeJS. Make sure to replace `NODE_MAJOR` by the version number of the latest node runtime.
```
Comment (RoRoJ) [inline tutorials/install-parse-server/index.mdx:65]:
```suggestion
1. Add an external repository for the required version of NodeJS. Make sure to replace `NODE_MAJOR` with the version number of the latest node runtime.
```

File: tutorials/jitsi-ubuntu-jammy-jellyfish/index.mdx [hunk 4]
```diff
@@ -30,7 +33,7 @@ The tool provides features like:
 </Message>
 
 <Message type="note">
-  This tutorial shows the installation of Jitsi Meet on Ubuntu Jammy Jellyfish. If you prefer the installation of Jitsi Meet on Ubuntu Bionic Beaver, follow [this tutorial](/tutorials/jitsi-ubuntu-bionic).
+  This tutorial shows the installation of Jitsi Meet on Ubuntu Jammy Jellyfish. If you prefer the installation of Jitsi Meet on Ubuntu Bionic Beaver, follow [the tutorial for installing Jitsi Meet on Ubuntu Bionic Beaver](/tutorials/jitsi-ubuntu-bionic).
```
Comment (RoRoJ) [inline tutorials/jitsi-ubuntu-jammy-jellyfish/index.mdx:36]:
```suggestion
  This tutorial shows the installation of Jitsi Meet on Ubuntu Jammy Jellyfish. If you would prefer to install Jitsi Meet on Ubuntu Bionic Beaver, follow [the tutorial for installing Jitsi Meet on Ubuntu Bionic Beaver](/tutorials/jitsi-ubuntu-bionic).
```

File: network/public-gateways/how-to/delete-a-public-gateway.mdx [hunk 5]
```diff
@@ -23,7 +23,7 @@ This page shows you how to delete a [Public Gateway](/network/public-gateways/co
 </Message>
 
 1. Click **Public Gateways** in the **Network** section of the side menu.
-2. Click <Icon name="more" /> next to the Public Gateway you wish to delete, and select **Delete** from the drop-down menu.
+2. Click "more" icon (<Icon name="more" />) next to the Public Gateway you wish to delete, and select **Delete** from the drop-down menu.
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply network/public-gateways/how-to/delete-a-public-gateway.mdx]: We should use this `Click <Icon name="more" /> next`

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**